README.md

```diff
@@ -92,14 +92,25 @@ For projects hosted by [GitLab](https://gitlab.com) reference [Microsoft.SourceL
 
 ### Bitbucket.org
 
-For projects hosted on [Bitbucket.org](https://bitbucket.org) in git repositories reference [Microsoft.SourceLink.Bitbucket.Git](https://www.nuget.org/packages/Microsoft.SourceLink.Bitbucket.Git) package: 
+For projects hosted on [Bitbucket.org](https://bitbucket.org) in git repositories reference [Microsoft.SourceLink.Bitbucket.Git](https://www.nuget.org/packages/Microsoft.SourceLink.Bitbucket.Git) package:
 
 ```xml
 <ItemGroup>
   <PackageReference Include="Microsoft.SourceLink.Bitbucket.Git" Version="1.0.0-beta2-18618-05" PrivateAssets="All"/>
 </ItemGroup>
 ```
 
+For self-hosted Bitbucket projects reference [Microsoft.SourceLink.Bitbucket.Git](https://www.nuget.org/packages/Microsoft.SourceLink.Bitbucket.Git) package and add Bitbucket host configuration.
+Additionally there are two optional attributes with default values: 
+EnterpriseEdition="true" - flag whether it is Enterprise Edtition or Cloud Edition
```

Comment:
>Edtition [](start = 57, length = 8)

typo: `Edtition`

Reply:
I don't think markdown will format these as you intended - on separate lines.

---
In reply to: [283563555](https://github.com/dotnet/sourcelink/pull/246#discussion_r283563555) [](ancestors = 283563555)

Reply:
Let's include the EnterpriseEdition and Version parameters to the example below.


---
In reply to: [283563725](https://github.com/dotnet/sourcelink/pull/246#discussion_r283563725) [](ancestors = 283563725,283563555)